### PR TITLE
Fix/build

### PR DESCRIPTION
**Pull request by Rorck**
Removed dead code causing build errors.

Changed the dynamic import of routers in the `index.ts` to static imports, which were also causing build errors.

Now the catalog-api can be build and can be with started with `pnpm run build && pnpm run start` or with `docker compose up`.

### CODE REVIEW

File: src/routes/index.ts [hunk 1]
```diff
@@ -1,52 +1,100 @@
...
-import path from "path";
-import { Application } from "express";
+import e, { Application } from "express";
 
-const importRouter = async (routerPath: string) => {
-  const routerModule = await import(routerPath);
-  return routerModule.default;
-};
+// public routers
+import auth from "./public/v1/auth";
```
Comment (Visions-Nicolas) [inline src/routes/index.ts:4]: I'm not sure if manually importing the required routes is a better approach, as the previous router handled this automatically. Did you encounter any issues after deleting the libs/loaders/routes.ts file and removing its import from the server.ts file?

Comment (Rorck) [reply src/routes/index.ts:4]: Yes, the automatic import was causing the following runtime error:

```
catalog-api          | /usr/src/app/dist/src/routes/private/v1/dataResources.private.router.d.ts:1
catalog-api          | import { Router } from "express";
catalog-api          | ^^^^^^
catalog-api          | 
catalog-api          | SyntaxError: Cannot use import statement outside a module
catalog-api          |     at wrapSafe (node:internal/modules/cjs/loader:1378:20)
catalog-api          |     at Module._compile (node:internal/modules/cjs/loader:1428:41)
catalog-api          |     at Module._extensions..js (node:internal/modules/cjs/loader:1548:10)
catalog-api          |     at Module.load (node:internal/modules/cjs/loader:1288:32)
catalog-api          |     at Module._load (node:internal/modules/cjs/loader:1104:12)
catalog-api          |     at Module.require (node:internal/modules/cjs/loader:1311:19)
catalog-api          |     at require (node:internal/modules/helpers:179:18)
catalog-api          |     at /usr/src/app/dist/src/routes/index.js:34:92
catalog-api          |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
catalog-api          |     at async importRouter (/usr/src/app/dist/src/routes/index.js:34:26)
catalog-api          | 
catalog-api          | Node.js v20.17.0
catalog-api          | ELIFECYCLE Command failed with exit code 1.
```

Comment (gonczy) [reply src/routes/index.ts:4]: @Visions-Nicolas @FelixBole could you please close this to have a public version without build errors?

Comment (Visions-Nicolas) [reply src/routes/index.ts:4]: After removing the dead code, the dynamic router is working well. Could you revert the router to the previous version ?

**Review by FelixBole — APPROVED** [review]
Verfieid on lts & node 18, both were indeed failing.
We'll take this path for now and investigate why the dynamic imports fail.

**Pull request merged**